packages/client/src/comp/Navbar.tsx

```diff
@@ -163,8 +163,7 @@ const LocaleToggle = styled.div<NavExpanded>`
     width: var(--rowWidth);
 
     position: fixed;
-    /* Creates the effect that the dropdown is touching/merged with the navbar */
-    top: calc(var(--height) * 0.97);
+    top: var(--height);
```

Comment:
Discovered that the issue that I was having (1px between the dropdown and navbar) only happened in simulators, so it is safe to remove this hack.

Reply:
LGTM